README.md

```diff
@@ -0,0 +1,125 @@
+o-spacing [![Circle CI](https://circleci.com/gh/Financial-Times/o-spacing/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-spacing/tree/master)[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
+=================
+
+_Maintain consistent spaces across components and projects in accordance to our baseline grid and design guidelines._
+
+- [Markup](#markup)
+- [CSS Custom Properties](#css-custom-properties)
+- [Sass](#sass)
+- [Troubleshooting](#troubleshooting)
+- [Contributing](#contributing)
+- [Migration](#migration)
+- [Contact](#contact)
+- [Licence](#licence)
+
+For [Build Service](https://www.ft.com/__origami/service/build/v2/) users, `o-spacing` provides [CSS classes](#markup) for vertical space and [CSS Custom Properties (CSS Variables)](#css-custom-properties) for other usecases. For Sass users `o-spacing` also provides a number of [Sass functions and mixins](#sass) for applying space to a project.
+
+### Spaces
+
+![named-spaces](https://user-images.githubusercontent.com/10405691/57918050-e9381780-788d-11e9-8310-ba5053c0c84a.png)
```

Comment:
Where does the original image live? Should it be in the repo?

Reply:
It lives in the UXD Toolkit Sketch file. I'll add a link to it https://uxd.in.ft.com/standards/

Reply:
Sorry linked to the wrong site before.